src/Microsoft.AspNetCore.Mvc.Razor.Extensions/NamespaceDirective.cs

```diff
@@ -92,32 +77,29 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument, DocumentInte
         //
         // In the event that these two source either don't have FileNames set or don't follow a coherent hierarchy,
         // we will just use the namespace verbatim.
-        internal static bool TryComputeNamespace(string source, DirectiveIntermediateNode directive, out string @namespace)
+        internal static string GetNamespace(string source, DirectiveIntermediateNode directive)
```

Comment:
This is a good improvement, not sure what I was smoking when I wrote this code, but I wish I still had some 😆